scripts/migrations/write_prefs_to_store.py

```diff
@@ -112,6 +112,7 @@ def main(args):
             f"Begin writing batch of {len(affected_pref_keys)} preferences to store..."
         )
         cur_batch = affected_pref_keys[:1000]
+        affected_pref_keys = affected_pref_keys[1000:]
```

Comment:
This is happening in a loop that doesn't complete until the `affected_pref_keys` list is empty, so it's important to remove each batch of entries from the list.